site-packages/integralstor_utils/config.py

```diff
@@ -115,6 +117,18 @@ def get_minimum_user_uid_gid(type='user'):
         return id, None
 
 
+def get_normal_users():
+    try:
+        ret, err = command.get_command_output("awk -F'[/:]' '{if ($3 >= 1000 && $3 < 65534) print $1}' /etc/passwd")
```

Comment:
Does the 1000 refer to the minimum normal user id? If so, can it be replaced by get_minimum_user_uid_gid() in config.py? That way, if we want to change it later, it is restricted to one change?